documentation/man/features/env.rst

```diff
@@ -48,6 +48,9 @@ OPTIONS
 -l, \--list:
   It shows all envs created via `\--create` option.
 
+-n, \--no-env:
+  Allow users to "exit" the env feature.
```

Comment:
After reading this, I think that `--exit-env` is a better name.

Reply:
Also, explain what exit an env means. Explain the behavior as we discussed in the https://github.com/kworkflow/kworkflow/issues/746 .